outputs.tf

```diff
@@ -8,6 +8,12 @@ output "instance_id" {
   value       = scaleway_rdb_instance.this.id
 }
 
+output "instance_ip" {
+  description = "Database instance IP"
+  sensitive   = true
```

Comment:
I don't see the point of having the instance IP marked as sensitive?

Reply:
You're right it's a mistake on my part.
I fixed that.